src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/Services/SemrushService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+
+using Umbraco.Cms.Integrations.SEO.SemrushTools.Models.Dtos;
+using Umbraco.Core.Services;
+
+namespace Umbraco.Cms.Integrations.SEO.SemrushTools.Services
+{
+    public class SemrushService: ISemrushService<TokenDto>
```

Comment:
If you following the suggestion above, this would become: `SemrushTokenService: ISemrushTokenService`.

Reply:
Item addressed.